New package: ImGuiTestEngine v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: ImGuiTestEngine
- Repository: https://github.com/JuliaImGui/ImGuiTestEngine.jl
- Created by: @JamesWrigley
- Version: v0.1.0
- Commit: 73b511c9f9a0baff4d0b2fea38da5621d6431e2a
- Reviewed by: @JamesWrigley
- Reference: https://github.com/JuliaImGui/ImGuiTestEngine.jl/commit/73b511c9f9a0baff4d0b2fea38da5621d6431e2a#commitcomment-143639005
- Description: Julia bindings for the Dear ImGui test engine
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6d19525416556cb12809aede10096b4e8b275c80d772eb49df1b96d31037b5b57b3c657e6bf4f9f623c5374bec503db2ef7a77be4ce4f7d86a7f6a4ad0dabc9f6ed5f921ab87bedba48dba937405dee5c6a2db1932116aa06661a55bd4916e825bdf94eaa0dfaa9e429ffa770b623274da392ad126a4c3b6b781b0e090956d360070e96c39dab13ef77e7c7ed19cff28f52e25e2e3ba9a4e932fe99bff084b1d2aada28f9c94eecf1b6749533d9d34f73 -->

JamesWrigley commented:
Unfortunately I think this will need a manual merge because the package requires Julia 1.11: https://github.com/JuliaRegistries/General?tab=readme-ov-file#can-my-package-be-registered-if-it-requires-a-version-of-julia-that-is-not-yet-released

[noblock]

JuliaRegistrator committed:
New package: ImGuiTestEngine v0.1.0

UUID: 464e2eba-0a11-4ed3-b274-413caa1a1cca
Repo: https://github.com/JuliaImGui/ImGuiTestEngine.jl.git
Tree: 9bbbeda1ddec398295fc03c4184a492a0b7262b6

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
